src/lib/plugins/useItem.js

```diff
@@ -27,10 +40,24 @@ module.exports.server = (serv, { version }) => {
         }
       }
     } else {
-      serv.onItemPlace('spawn_egg', ({ heldItem, player, placedPosition }) => {
-        serv.spawnMob(getEntID(heldItem.nbt.value.EntityTag.value.id.value), player.world, placedPosition)
-        return { id: -1, data: 0 }
-      })
+      if (version === '1.12.2') {
+        serv.onItemPlace('spawn_egg', ({ item, player, placedPosition }) => {
+          serv.spawnMob(getEntIDmc(item.nbt.value.EntityTag.value.id.value), player.world, placedPosition)
+          return { id: -1, data: 0 }
+        })
+      } else {
+        if (version === '1.8.8') {
```

Comment:
To check the version you need to use the supportFeature system (it must also work for other 1.8 versions (snapshots etc..)

Basically, you create an entry in https://github.com/PrismarineJS/flying-squid/blob/master/src/lib/features.json#L63
And check with
```js
if (serv.supportFeature('entityMCPrefixed')) {
```

Reply:
Applying...